### ISSUE TITLE
Lizzie passes but does not resign?

### ISSUE THREAD

**Issue by sergio0p** [featurecat/lizzie#648]
I'm using Lizzie with Leela-zero and when playing handicap games, sometimes, I guess when it falls behind, Lizzie keeps passing and never resigns. 

Leela-zero seems to be running fine. When I run autogtp, it runs the self-play games ok. When I use Lizzie without handicaps and lose, also there are no other problems.

First time using this, I searched for "passes" but did not find any previous related issue.

**Comment by featurecat** [comment]
Right... I think actually Lizzie is a bit bugged when it comes to playing handicap games. I'm not sure exactly what is the cause of the problem. Sometimes it even believes a stone on the board doesn't exist, or hallucinates stones that aren't on the board, especially during handicap games. @zsalch could you please investigate the playing-mode issues?

**Comment by 84nobby** [comment]
I guess you are still using Lizzie 0.6. 
I recommend you upgrade to Lizzie 0.7.1 which is already released in this Github. 
Handcapped Game can be done without any problems and freezing.

**Comment by sergio0p** [comment]
@84nobby If the README file is correct I am using Lizzie 0.7, I will install 0.72 and comment later.
